fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarController.kt

```diff
@@ -217,7 +217,7 @@ class DefaultBrowserToolbarController(
         )
     }
 
-    override fun handleShoppingCfrDismiss() {
```

Comment:
Can't comment there but `updateShippingCfrSettings()` is still called in `handleShoppingCfrActionClick()`, won't that lead to twice incrementing the counter - once on display and once on click should the user click as well. In that case, we should remove it from there.